policies/pull-request-review-and-merge.md

```diff
@@ -0,0 +1,65 @@
+# Policy on pull request review and merging in the Astropy Project
+
+## Astropy core
+
+In the astropy core package the requirements are as follows:
+
+- At least one contributor with relevant expertise must review the pull request
+  in detail.
+  - New or updated code is logically correct and achieves the desired effect of
+    either fixing a bug or making an enhancement.
+  - New tests or modifications to tests are adequate to cover the code changes.
+    Tests for edge cases exist, if applicable.
```

Comment:
Should there be something making this a bit softer like:
```suggestion
    Tests for edge cases exist, if applicable and judged to be important enough.
```
so that it doesn't sound like we are demanding all possible edge cases always be accounted for